omero/sysadmins/version-requirements.rst

```diff
@@ -2,26 +2,18 @@
 Version requirements
 ********************
 
-Summary of changes for OMERO 5.4 and provisional changes for 5.5
-================================================================
+Introduction
+============
 
 Criteria for what is considered to be supportable includes whether
 support by both upstream developers and operating system distributions
-will be available for the lifetime of the 5.4 release (including
+will be available for the lifetime of the |version| release (including
 security support), and also upon our resources allocated to CI and
 testing. If we are not actively testing it, we **cannot** claim it is
 supported or functional. Software components must be provided and
 supported either by an operating system distribution or their original
 developers.
 
-This section contains a summary of the changes made to the minimum
-version requirements for the 5.4 release and also **possible** changes for
```

Comment:
As @sbesson mentioned, I'd think we either remove the OMERO 5.6 column, or add a versino of text back in to make the tentativeness clear.